src/textarea/src/Textarea.js

```diff
@@ -26,14 +26,13 @@ const Textarea = memo(
       ...restProps
     } = props
 
-    const themedClassName = useInputAppearance()
+    const { className: themedClassName, boxProps } = useInputAppearance()
 
     return (
-      <Text
+      <Box
         is="textarea"
         ref={ref}
         className={cx(themedClassName, className)}
-        size={300}
         width={width}
```

Comment:
What do you think about pushing `height` out here like we did in `<TextInput />`?

Reply:
I think you want to be able to control the height of the textarea as opposed to an input. Feels like we can leave this one

Reply:
But you can still specify height? Won't it pass through via restProps?

Reply:
yeah, suppose one difference is just the default `height` of the textarea as opposed to the other inputs